x86_64/python-fenics-ufl/cactus.yaml

```diff
@@ -0,0 +1 @@
+../../template/x86_64-simple.yaml
```

Comment:
This one should use the x86_64-python template.

Reply:
Thanks, done with `ln -s ../../template/x86_64-python cactus.yaml`.